code/modules/jobs/job_types/trusted_players/association/director.dm

```diff
@@ -37,6 +37,7 @@ GLOBAL_LIST_INIT(association_jobs, list(
 		/datum/job/sottocapo,
 		/datum/job/grandinquis,
 		/datum/job/kurocaptain,
+		/datum/job/DeadRabbitsBoss,
```

Comment:
Minor thing, this file was moved to association/col_association or some shit like that